xapian-applications/omega/docs/postprocessing.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Log Data Postprocessing
+=========================
+
+Log data includes a) data logged from the search results page rendered by
+Omega query template and, b) clicks recorded when a document link is
+clicked on the results page.
+
+Further processing of this raw data is handled by the ``postprocess.py``
```

Comment:
There's probably a better name for this than just "postprocess", but I can't think of one right now.